applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -789,8 +894,13 @@ static void unicast_client_location_cb(struct bt_conn *conn, enum bt_audio_dir d
 	struct le_audio_unicast_server *unicast_server =
 		&unicast_servers[idx.lvl1][idx.lvl2][idx.lvl3];
 
-	if ((loc & BT_AUDIO_LOCATION_FRONT_LEFT) || (loc & BT_AUDIO_LOCATION_SIDE_LEFT) ||
-	    (loc == BT_AUDIO_LOCATION_MONO_AUDIO)) {
+	if (loc & BT_AUDIO_LOCATION_FRONT_LEFT && loc & BT_AUDIO_LOCATION_FRONT_RIGHT) {
+		LOG_INF("Both front left and right channel locations are set, stereo device found");
+		unicast_server->location =
+			BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT;
+		unicast_server->ch_name = "STEREO";
+	} else if ((loc & BT_AUDIO_LOCATION_FRONT_LEFT) || (loc & BT_AUDIO_LOCATION_SIDE_LEFT) ||
+		   (loc == BT_AUDIO_LOCATION_MONO_AUDIO)) {
 		unicast_server->location = BT_AUDIO_LOCATION_FRONT_LEFT;
 		unicast_server->ch_name = "LEFT";
 
```

Comment:
Might be overlooking something here, but if you encounter a server that only supports mono, we don't support that? 

If `loc == 0x0000` that is also mono, but looks like it would hit the `LOG_WRN("Channel location not supported: %d", loc);` clause here?